samples/text/text.cpp

```diff
@@ -1668,6 +1698,35 @@ void MyFrame::OnFileLoad(wxCommandEvent& WXUNUSED(event))
     }
 }
 
+void MyFrame::OnFileSaveRtf(wxCommandEvent& WXUNUSED(event))
+{
+    if (m_panel->m_textrich->SaveFile("dummy.rtf", wxTEXT_TYPE_RTF))
+    {
+#if wxUSE_FILE
+        // verify that the file length is correct
+        wxFile file("dummy.rtf");
+        wxLogStatus(this,
+            "Successfully saved file (text len = %lu, file size = %ld)",
+            (unsigned long)m_panel->m_textrich->GetValue().length(),
+            (long)file.Length());
```

Comment:
We should be able to avoid casts here:
```suggestion
            "Successfully saved file (text len = %zu, file size = %lld)",
            m_panel->m_textrich->GetValue().length(),
            file.Length());
```

Reply:
Got it